### ISSUE TITLE
[icerec] Incremental Release V05-00-06 (Trac #1918)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1910]
This is a tracking ticket for icerec V05-00-06.

See comments for changes that need to be included.

Migrated from https://code.icecube.wisc.edu/ticket/1918
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:30", 
    "description": "This is a tracking ticket for icerec V05-00-06.\n\nSee comments for changes that need to be included.", 
    "reporter": "joertlin", 
    "cc": "david.schultz, kjmeagher", 
    "resolution": "fixed", 
    "_ts": "1550067210114669", 
    "component": "combo reconstruction", 
    "summary": "[icerec] Incremental Release V05-00-06", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-12-02T17:17:28", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161202T17:18:53: joertlin commented:

> I3EventHeader SubEventId fix: http://code.icecube.wisc.edu/projects/icecube/changeset/151909/IceCube
> 
> '''Not''' urgent.

**Comment by icecube-trac** [comment]
Trac update at 20161206T16:58:41: 
* olivas changed cc from "david.schultz" to "david.schultz, kjmeagher"
* olivas changed milestone from "" to "December 2016"

**Comment by icecube-trac** [comment]
Trac update at 20170118T19:46:15: 
* olivas changed resolution from "" to "fixed"
* olivas changed status from "new" to "closed"
* olivas commented:

> In changeset 152799:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="152799"
> using the bugfix release of dataclasses.  closes #1910.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:30: 
* olivas changed milestone from "December 2016" to ""
* olivas commented:

> Milestone December 2016 deleted
